librz/bin/format/java/class_bin.c

```diff
@@ -713,10 +714,15 @@ RZ_API void rz_bin_java_class_as_source_code(RzBinJavaClass *bin, RzStrBuf *sb)
 					rz_str_replace_char(tmp, '/', '.');
 
 					char *ptr = strchr(dem, '(');
-					*(ptr - 1) = 0;
-					rz_strbuf_append(sb, dem);
-					rz_strbuf_appendf(sb, "%s", tmp);
-					rz_strbuf_append(sb, ptr);
+					if (ptr) {
+						*(ptr - 1) = 0;
+						rz_strbuf_append(sb, dem);
```

Comment:
just do dem+tmp before the `if` and then add the `rz_strbuf_append(sb, ptr);` if ptr is not null.

Reply:
No. i'm modifying some data. i can't. already tried and it failed. this is why i had to force push